pkg/network/iperf/iperf.go

```diff
@@ -0,0 +1,39 @@
+package iperf
+
+import (
+	"os/exec"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/zos/pkg/zinit"
+)
+
+const (
+	zinitService = "iperf"
+)
+
+// Ensure creates an iperf zinit service and monitors it
+func Ensure(z *zinit.Client) error {
+	if _, err := z.Status(zinitService); err == nil {
+		return nil
+	}
+
+	_, err := exec.LookPath("iperf")
+	if err != nil {
+		return err
+	}
+
+	cmd := `ip netns exec public iperf -s`
```

Comment:
as discussed, may be it's better to add the port explicitly in the command line. (from a constant) that same constant need to be reserved in networkd as i showed you over telegram